org.civicrm.reqangular/test/services/crm-ang.service.spec.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-env amd, jasmine */
+
+(function (CRM) {
+  define([
+    'common/angular',
+    'common/angularMocks',
+    'common/services/crm-ang.service'
+  ], function () {
+    'use strict';
+
+    describe('crmAngService', function () {
+      var crmAngService;
+
+      beforeEach(module('common.services'));
+      beforeEach(inject(function (_crmAngService_) {
+        crmAngService = _crmAngService_;
+      }));
+
+      describe('loadForm()', function () {
+        var result;
+        var url = '/some-url';
+        var options = { optionKey: 'optionValue' };
+        var originalResult = 'jQuery object';
+
+        beforeEach(function () {
+          CRM.loadForm = function () {};
```

Comment:
Can't you use [`.createSpy()`](https://jasmine.github.io/2.0/introduction.html#section-Spies:_%3Ccode%3EcreateSpy%3C/code%3E)?

Reply:
Done.